flashinfer/fp4_quantization.py

```diff
@@ -299,7 +345,7 @@ def fp4_quantize(
     return x_q, sf
 
 
-def nvfp4_block_scale_interleave(unswizzled_sf: torch.Tensor) -> torch.Tensor:
+def block_scale_interleave(unswizzled_sf: torch.Tensor) -> torch.Tensor:
```

Comment:
I would suggest also keeping `nvfp4_block_scale_interleave` for backward compatibility, as mentioned by @aleozlx 

```python
nvfp4_block_scale_interleave = block_scale_interleave
```